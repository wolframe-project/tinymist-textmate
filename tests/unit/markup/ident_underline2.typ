_#a_b _
