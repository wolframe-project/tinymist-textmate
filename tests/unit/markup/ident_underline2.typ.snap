>_#a_b _
#^ source.typst
# ^ source.typst variable.other.readwrite.hash.typst
#  ^^^ source.typst variable.other.readwrite.typst
#     ^^^ source.typst
>